lib/mini_profiler/version.rb

```diff
@@ -1,5 +1,5 @@
 module Rack
   class MiniProfiler
-    VERSION = '0.10.5'
+    VERSION = '0.10.5-olivervbk'
```

Comment:
oops this version thing seems to have creeped in, can you remove?

Reply:
I'm terribly sorry. I forgot to create a separate branch for the fix.
I've created a new PR: https://github.com/MiniProfiler/rack-mini-profiler/pull/311